source/templates/conditionals.md

```diff
@@ -1,7 +1,49 @@
-Sometimes you may only want to display part of your template if a property
-exists.
+Conditional statements bring a minimum of logic into Ember templating.
+Statements like `if` and `unless` are implemented as built-in helpers. Helpers
+can be invoked three ways.
```

Comment:
Oh, I'm starting to understand what you've done with this guide - it now about both the specific case of the conditionals helpers, as well as the general case of all helpers. I think this works, but I think it's a little confusing without stating at the beginning somewhere that the purpose of the page is both to introduce conditionals (which is the title) **and** to give an overview of how helpers work in general.